src/client/api/tests/job_tests.c

```diff
@@ -99,24 +101,22 @@ teardown_job_mocks(void **state)
 }
 
 static int
-craft_jobid(char **jobid, char *nodename , pid_t pid)
+craft_jobid(char **jobid, char *nodename, pid_t pid)
```

Comment:
Since this function now asserts on error, it only returns zero. Could just make it a void function now.